geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -56,14 +56,15 @@ class SelectionMapFragment(
     @Inject
     lateinit var permissionsChecker: PermissionsChecker
 
-    private val selectedFeatureViewModel by viewModels<SelectedFeatureViewModel>()
+    private val selectedItemViewModel by viewModels<SelectedItemViewModel>()
 
     private lateinit var map: MapFragment
     private lateinit var summarySheetBehavior: BottomSheetBehavior<*>
     private lateinit var summarySheet: SelectionSummarySheet
     private lateinit var bottomSheetCallback: BottomSheetCallback
 
     private val itemsByFeatureId: MutableMap<Int, MappableSelectItem> = mutableMapOf()
+    private val featureIdsByItemId: MutableMap<Long, Int> = mutableMapOf()
```

Comment:
It maybe feels excessive to have two `Map` indexes here, but otherwise we'd need to do linear searches on several click events, and we build these maps in a loop we're already performing.